front/app/containers/UsersEditPage/DeletionDialog.tsx

```diff
@@ -51,55 +45,26 @@ const ButtonsContainer = styled.div`
   display: flex;
 `;
 
-interface InputProps {
+interface Props {
   closeDialog: () => void;
 }
 
-interface DataProps {
-  tenant: GetAppConfigurationChildProps;
-  authUser: GetAuthUserChildProps;
-}
-
-interface Props extends InputProps, DataProps {}
-
-interface State {
-  error: boolean;
-  processing: boolean;
-}
-
-class DeletionDialog extends PureComponent<Props & InjectedIntlProps, State> {
-  constructor(props) {
-    super(props);
-    this.state = {
-      error: false,
-      processing: false,
-    };
-  }
+const DeletionDialog = ({
+  closeDialog,
+  intl: { formatMessage },
+}: Props & InjectedIntlProps) => {
+  const appConfiguration = useAppConfiguration();
 
-  deleteProfile = () => {
+  const deleteProfile = () => {
     signOutAndDeleteAccountPart1();
-    this.props.closeDialog();
+    closeDialog();
   };
 
-  render() {
-    const {
-      tenant,
-      intl: { formatMessage },
-    } = this.props;
-    const { processing, error } = this.state;
-    const tenantLogo = !isNilOrError(tenant)
-      ? get(
-          tenant.attributes.logo,
-          'medium',
-          get(tenant.attributes.logo, 'small')
-        )
-      : null;
-
+  if (!isNilOrError(appConfiguration)) {
+    const logo = appConfiguration.data.attributes.logo?.medium || null;
```

Comment:
I looked this up because I was curious but I think the `|| null` is unnecessary here with the optional chaining operator, unless we specifically want it to be null rather than undefined. If the app configuration has a data property, which has an attributes property, using the `?.` operator on logo should return undefined if `logo` isn't present, and if it is present but `medium` isn't a property on `logo`, it will also return undefined. It's totally fine as-is though, just checked the details out for my own knowledge.

Reply:
You're right ✅